sabnzbd/utils/certgen.py

```diff
@@ -53,20 +44,33 @@ def generate_key(key_size=2048, output_file='key.pem'):
             encoding=serialization.Encoding.PEM,
             format=serialization.PrivateFormat.TraditionalOpenSSL,
             encryption_algorithm=serialization.NoEncryption()
+            # encryption_algorithm=serialization.BestAvailableEncryption(b"passphrase")
         ))
 
     return private_key
 
 
-def generate_local_cert(private_key, days_valid=356, output_file='cert.cert', LN='', ON='', CN=''):
+# Ported from cryptography docs/x509/tutorial.rst
+def generate_local_cert(private_key, days_valid=3560, output_file='cert.cert', LN=u'SABnzbd', ON=u'SABnzbd', CN=u'localhost'):
```

Comment:
certgen is ran for one of 3 scenarios..  we do on refresh.. we do on startup.. user can manually from the dir.  putting the defaults here ensures that all 3 cases result in the same cert.